registry/handlers/app.go

```diff
@@ -43,7 +43,7 @@ const randomSecretSize = 32
 type App struct {
 	context.Context
 
-	Config configuration.Configuration
+	Config *configuration.Configuration
```

Comment:
I changed it to a pointer because I saw that we weren't doing a deep copy, and it seemed like no copy was better than a partial one. There are a number of maps and slices inside the config that would end up being shared even if `Config` was not a pointer.

I can change this back, since it's not really related to the PR. But my personal feeling is that we should avoid doing a shallow copy.